README.md

```diff
@@ -135,6 +135,20 @@ Furthermore, all python code must run through [pyright](https://github.com/micro
 without errors. Pyright can be run on all staged files through the
 makefile using `make pyright`.
 
+> [!IMPORTANT]
+>
+> ## Experimental Pyright Features
```

Comment:
This looks like it's being promoted to a top-level heading in the important block